room.go

```diff
@@ -545,6 +549,28 @@ func (r *Room) Simulate(scenario SimulateScenario) {
 		}
 		r.engine.client.SendRequest(req)
 		r.engine.client.OnLeave(&livekit.LeaveRequest{CanReconnect: true, Reason: livekit.DisconnectReason_CLIENT_INITIATED})
+	case SimulateSpeakerUpdate:
+		r.engine.client.SendRequest(&livekit.SignalRequest{
+			Message: &livekit.SignalRequest_Simulate{
+				Simulate: &livekit.SimulateScenario{
+					Scenario: &livekit.SimulateScenario_SpeakerUpdate{
+						SpeakerUpdate: SimulateSpeakerUpdateInterval,
+					},
+				},
+			},
+		})
+	case SimulateMigration:
+		req := &livekit.SignalRequest{
+			Message: &livekit.SignalRequest_Simulate{
+				Simulate: &livekit.SimulateScenario{
+					Scenario: &livekit.SimulateScenario_Migration{
+						Migration: true,
+					},
+				},
+			},
+		}
+		r.engine.client.SendRequest(req)
+		r.engine.client.OnLeave(&livekit.LeaveRequest{CanReconnect: true, Reason: livekit.DisconnectReason_CLIENT_INITIATED})
```

Comment:
Don't need call OnLeave here, as the server side will close signal connection at the migration case. And I think go-sdk don't support migration since pion don't support dtls restart.